src/components/shared/SearchInput.js

```diff
@@ -115,7 +119,7 @@ class SearchInput extends Component {
         placeholder,
         value: query,
         spellCheck: false,
-        ref: "input"
+        ref: c => (this.input = c)
```

Comment:
is this the preferred style?

perhaps? `ref: c => (this.$input = c)`

it's funny, i prefer refs because their namespaced, but i'll support anything.

Also this style is fine w/ me too
https://github.com/devtools-html/debugger.html/blob/master/src/components/shared/Autocomplete.js#L51-L53